core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -1332,22 +1341,21 @@ public void onConfigurationChanged(Configuration newConfig) {
     tabRecyclerView.setAdapter(tabsAdapter);
   }
 
-  private void searchForTitle(String title) {
+  private void searchForTitle(String title, Boolean newTab) {
     String articleUrl;
 
     if (title.startsWith("A/")) {
       articleUrl = title;
     } else {
       articleUrl = zimReaderContainer.getPageUrlFromTitle(title);
     }
-    openArticle(articleUrl);
+     openArticle(articleUrl, newTab);
```

Comment:
suspicious space added here, I would imagine this no longer adheres to our codestyle, press ctrl+alt+l to reformat the file you are in, I personally have a macro bound to ctrl+s that formats/saves/indents/optimises